README.md

```diff
@@ -1,5 +1,102 @@
-# thorvg.flutter
-<p align="center">
-  <img width="800" height="auto" src="https://github.com/thorvg/thorvg/blob/main/res/logo/512/thorvg-banner.png">
-</p>
-ThorVG for Flutter System
+# ThorVG for Flutter
+
+This package provides [ThorVG](https://github.com/thorvg/thorvg) runtime for Flutter, including efficient Lottie animation support based on a native API.
+
+> Currently, we only support Animation(Lottie) feature in this package.
+
+## Supported Platforms
+
+| Platform | Architecture |
+| ------------- | ------------- |
+| Android | arm64-v8a, armeabi-v7a, x86_64 |
+| iOS | arm64, x86_64, x86_64(simulator) |
+
+## Usage
+
+### Animation
+Lottie implementation aims to maintain the same interface as `lottie-flutter`. If you are currently using them, you can utilize the code by simply replacing the import statement with `import 'package:thorvg/thorvg.dart'`.
+
+```dart
+import 'package:thorvg/thorvg.dart';
+// ...
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: Column(
+          children: [
+            // Load a Lottie animation from the assets
+            Lottie.asset('assets/lottie/dancing_star.json'),
+
+            // Load a Lottie animation from a url
+            Lottie.network(
+              'https://lottie.host/6d7dd6e2-ab92-4e98-826a-2f8430768886/NGnHQ6brWA.json'
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+```
+
+## Generate Flutter binding
+
+If you change binding interface in these files
+- `tvgFlutterLottieAnimation.h`
+- `tvgFlutterLottieAnimation.cpp`
+
+You always have to run following script:
+
+```sh
+flutter pub run ffigen --config ffigen.yaml
```

Comment:
Before doing this you will probably need to do a `flutter pub get` first.

Reply:
@JSUYA Added the script!